Specs/DateTime/Portuguese/DateTimeModel.json

```diff
@@ -1825,5 +1825,82 @@
         "End": 34
       }
     ]
+  },
+  {
+    "Input": "meu próximo salário será terça-feira dia 25",
+    "Context": {
+      "ReferenceDateTime": "2018-09-28T01:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "terça-feira dia 25",
+        "Start": 25,
+        "End": 42,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2018-09-25",
+              "type": "date",
+              "value": "2018-09-25"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "meu próximo salário será terça-feira dia vinte e cinco",
+    "Context": {
+      "ReferenceDateTime": "2018-09-28T01:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "terça-feira dia vinte e cinco",
+        "Start": 25,
+        "End": 53,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2018-09-25",
+              "type": "date",
+              "value": "2018-09-25"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "meu próximo salário será terça-feira 25",
+    "Context": {
+      "ReferenceDateTime": "2018-09-28T01:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "terça-feira 25",
+        "Start": 25,
+        "End": 38,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-WXX-2",
```

Comment:
While this PR solves extraction and resolution for the issue, it seems we have a timex issue to refine.
1- This timex is consistent with the example given in the issue for Spanish. Why does "terça-feira dia 25" return differently (also the French cases)? They should be consistent, as they are the same date reference.
If possible, the timex generation and resolution code should be re-used to avoid such inconsistencies. Please compare to the behaviour in English.
2- Once consistency is corrected, we may also consider if the day and day-of-week should both be added in the generalized timex (something like XXXX-XX-25-2). We'll take a closer look and open an issue as appropriate. This is outside the scope of this issue and PR.

Reply:
This happens also in English for the patterns "Tuesday the 25th" and "Tuesday 25", the first is extracted by WeekDayAndDayOfMothRegex and the second by WeekDayAndDayRegex and they follow different parsing steps which produce different results.

Reply:
In Spanish, "martes 25" is processed as "terça-feira 25"/"Tuesday 25". Instead, "martes el dia 25" is wrongly parsed as ForTheRegex instead of WeekDayAndDayOfMothRegex. This need to be corrected too.
So, in this PR, I will correct this Spanish behaviour and modify the timex returned by WeekDayAndDayOfMothRegex to be in line with that returned by WeekDayAndDayRegex.

Reply:
Makes sense. Please correct the Spanish behaviour to match the others. Thanks.

Reply:
We'll analyze the unification of regex forms and potential creation of a new form for this case and create a new issue, if necessary.

Reply:
conformed Spanish behavior to match the others